Explorer/Assets/DCL/AvatarRendering/Wearables/Components/Wearable.cs

```diff
@@ -4,18 +4,38 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using UnityEngine;
 
 //Removed all references to EmoteData in WearableItem
 namespace DCL.AvatarRendering.Wearables.Components
 {
     [Serializable]
     public class Wearable : IWearable
     {
+        private const string THUMBNAIL_DEFAULT_KEY = "thumbnail.png";
         public StreamableLoadingResult<SceneAssetBundleManifest>? ManifestResult { get; set; }
         public StreamableLoadingResult<WearableAsset>?[] WearableAssetResults { get; private set; } = new StreamableLoadingResult<WearableAsset>?[BodyShape.COUNT];
         public StreamableLoadingResult<WearableDTO> WearableDTO { get; set; }
+
+        public StreamableLoadingResult<Texture2D>? WearableThumbnail { get; set; }
         public bool IsLoading { get; set; } = true;
 
+        public string GetThumbnail()
+        {
+            string thumbnailHash = WearableDTO.Asset.metadata.thumbnail;
+
+            if (thumbnailHash == THUMBNAIL_DEFAULT_KEY)
+            {
+                for (var i = 0; i < WearableDTO.Asset.content.Length; i++)
```

Comment:
There is the same piece of code in `GetMainFileHash`:
```
for (var i = 0; i < WearableDTO.Asset.content.Length; i++)
            {
                WearableDTO.WearableContentDto wearableContentDto = WearableDTO.Asset.content[i];

                if (wearableContentDto.file.Equals(mainFileKey))
                {
                    hashToReturn = wearableContentDto.hash;
                    break;
                }
            }
```

Please consider introducing a method to remove repetition.